src/components/itemContextMenu.js

```diff
@@ -458,7 +459,7 @@ define(['apphost', 'globalize', 'connectionManager', 'itemHelper', 'appRouter',
                     getResolveFunction(resolve, id)();
                     break;
                 case 'artist':
-                    appRouter.showItem(item.ArtistItems[0].Id, item.ServerId);
+                    appRouter.showItem(item.AlbumArtists[0].Id, item.ServerId);
```

Comment:
@MrTimscampi We already check that in the function above, where we push options to the array that builds the context menu. If no Album Artist, no option will be displayed, so no way this would be executed.